coalib/output/ConsoleInteraction.py

```diff
@@ -548,20 +548,17 @@ def choose_action(console_printer, actions):
     :param actions:         Actions available to the user.
     :return:                Return choice of action of user.
     """
-    console_printer.print(format_lines(
-        "The following actions are applicable to this result:"))
-
     while True:
         console_printer.print(format_lines("*0: " +
-                                           "Apply no further actions."))
+                                           "Do nothing"))
```

Comment:
Im still unsure if we should try to create a list in the console interaction or still give options that end in period. is it possible for you to try and run and post here pictures for us to see how it looks if you make the descriptions:
a) bulleted, as in:
`` - do nothing
   - apply patch
``
b) still orders, as in:
``
Do nothing.
Apply patch.
``